packages/core-data/src/entity-types/index.ts

```diff
@@ -93,6 +98,7 @@ export interface PerPackageEntityRecords< C extends Context > {
 		| Page< C >
 		| Plugin< C >
 		| Post< C >
+		| PostStatusObject< C >
 		| PostRevision< C >
```

Comment:
Same as above, with regards to keeping the alphabetic order

Reply:
Done. Thanks